Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -958,60 +960,66 @@ public static int CalculateBackstabDamage(int damage, int backstabbingLevel)
-                {
-                    damage -= attacker.Level;
-                }
-            }
-            else if (AITarget.GetEnemyGroup() == DFCareer.EnemyGroups.Daedra)
+            if (target is EnemyEntity)
             {
-                if (((int)attacker.Career.DaedraAttackModifier & (int)DFCareer.AttackModifier.Bonus) != 0)
+                var enemyTarget = target as EnemyEntity;
+                if (enemyTarget.MobileEnemy.Affinity == MobileAffinity.Human)
                 {
-                    damage += attacker.Level;
+                    if (((int)attacker.Career.HumanoidAttackModifier & (int)DFCareer.AttackModifier.Bonus) != 0)
+                        damage += attacker.Level;
+                    if (((int)attacker.Career.HumanoidAttackModifier & (int)DFCareer.AttackModifier.Phobia) != 0)
+                        damage -= attacker.Level;
                 }
-                if (((int)attacker.Career.DaedraAttackModifier & (int)DFCareer.AttackModifier.Phobia) != 0)
+                else if (enemyTarget.GetEnemyGroup() == DFCareer.EnemyGroups.Undead)
                 {
-                    damage -= attacker.Level;
+                    if (((int)attacker.Career.UndeadAttackModifier & (int)DFCareer.AttackModifier.Bonus) != 0)
+                        damage += attacker.Level;
+                    if (((int)attacker.Career.UndeadAttackModifier & (int)DFCareer.AttackModifier.Phobia) != 0)
+                        damage -= attacker.Level;
                 }
-            }
-            else if (AITarget.GetEnemyGroup() == DFCareer.EnemyGroups.Humanoid)
-            {
-                if (((int)attacker.Career.HumanoidAttackModifier & (int)DFCareer.AttackModifier.Bonus) != 0)
+                else if (enemyTarget.GetEnemyGroup() == DFCareer.EnemyGroups.Daedra)
                 {
-                    damage += attacker.Level;
+                    if (((int)attacker.Career.DaedraAttackModifier & (int)DFCareer.AttackModifier.Bonus) != 0)
+                        damage += attacker.Level;
+                    if (((int)attacker.Career.DaedraAttackModifier & (int)DFCareer.AttackModifier.Phobia) != 0)
+                        damage -= attacker.Level;
                 }
-                if (((int)attacker.Career.HumanoidAttackModifier & (int)DFCareer.AttackModifier.Phobia) != 0)
+                else if (enemyTarget.GetEnemyGroup() == DFCareer.EnemyGroups.Animals)
                 {
-                    damage -= attacker.Level;
+                    if (((int)attacker.Career.AnimalsAttackModifier & (int)DFCareer.AttackModifier.Bonus) != 0)
+                        damage += attacker.Level;
+                    if (((int)attacker.Career.AnimalsAttackModifier & (int)DFCareer.AttackModifier.Phobia) != 0)
+                        damage -= attacker.Level;
                 }
             }
-            else if (AITarget.GetEnemyGroup() == DFCareer.EnemyGroups.Animals)
+            else // Assume target is player if not an enemy
```

Comment:
This seems reasonable, but I'd prefer an `else if (target is PlayerEntity)` just for safety though.